packages/augur-ui/src/modules/modal/containers/modal-claim-fees.ts

```diff
@@ -59,15 +64,49 @@ const mergeProps = (sP: any, dP: any, oP: any) => {
     const market = marketObj.marketObject;
     if (market) {
       const marketRep = formatAttoRep(marketObj.totalAmount);
+      const isForkingMarket = market.id === forkingMarket;
```

Comment:
But if `market` doesn't have an `id`, it would be undefined.
`undefined === undefined` would be true here, as opposed to `undefined === false`.

Reply:
if market doesn't have an `id` we have bigger problems. much much bigger problems

Reply:
but I see your point. I can make it null.